specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingRequest.json

```diff
@@ -0,0 +1,504 @@
+          "readOnly": true
+        },
+        "decisionReason": {
+          "description": "The reason to approve or decline the request.",
+          "type": "string"
+        },
+        "invoiceSectionId": {
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionName": {
+          "description": "The ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerId": {
+          "description": "The fully qualified ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerName": {
+          "description": "The ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerDisplayName": {
+          "description": "The name of the customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "The fully qualified ID that uniquely identifies a billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionName": {
+          "description": "The ID that uniquely identifies a billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionDisplayName": {
+          "description": "The name of the billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "justification": {
+          "description": "Justification for submitting request.",
+          "type": "string"
+        },
+        "recipients": {
+          "description": "The recipients of the billing request.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "scope": {
+          "description": "The billing scope for which the request was submitted (ex. '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}').",
+          "type": "string"
+        },
+        "status": {
+          "description": "Status of billing request.",
+          "enum": [
+            "Other",
+            "Pending",
+            "Approved",
+            "Declined",
+            "Cancelled",
+            "Completed",
+            "Expired"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BillingRequestStatus",
+            "modelAsString": true
+          }
+        },
+        "type": {
+          "description": "Type of billing request.",
+          "enum": [
+            "Other",
```

Comment:
We should allow UX to call for `Refund` or other internal types by passing the internal type in the `AdditionalInformation` bag.